File: src/main/webapp/src/app/models/activity-audio-channel.ts
export class ActivityAudioChannel {
    index: number = 0;
    volumeDb: number = -500;

    constructor(data?: any) {
        if(!data) {
        	return;
        }
        
        this.index = data.index;
        this.volumeDb = data.volumeDb;
    }
}

File: src/main/java/com/ascargon/rocketshow/api/ActivityAudioChannel.java
package com.ascargon.rocketshow.api;

import jakarta.xml.bind.annotation.XmlRootElement;
import java.util.ArrayList;
import java.util.List;

@XmlRootElement
public class ActivityAudioChannel {

    private int index = 0;

    private double volumeDb = 0;

    public int getIndex() {
        return index;
    }

    public void setIndex(int channel) {
        this.index = channel;
    }

    public double getVolumeDb() {
        return volumeDb;
    }

    public void setVolumeDb(double volumeDb) {
        this.volumeDb = volumeDb;
    }

}
